readme.md

```diff
@@ -224,28 +238,33 @@ var handleClickEvent = function(){
 button.on("click", handleClickEvent);
 ```
 
-Our favorite keyword: `this`.
-* **Q:** Where have we seen it before? Why do we use it?  
-
-Try this: insert the following line anywhere in our event listener...
+Try this-- insert the following line anywhere in our click-handler:
```

Comment:
i would add a *bit* about context here, don't have to go in depth, but talk at a high level what `this` is and tell them that we'll be going over it in more extensive detail on friday's class